src/uct/ib/base/ib_md.c

```diff
@@ -1191,6 +1182,20 @@ static ucs_status_t uct_ib_query_md_resources(uct_component_t *component,
     struct ibv_device **device_list;
     ucs_status_t status;
     int i, num_devices;
+    size_t memlock_limit;
+
+    status = ucs_sys_get_memlock_rlimit(&memlock_limit);
```

Comment:
it seems the test fails now , because a new warning is printed.
before this PR, the docker test did not even try to open IB device because uct_ib_device_is_accessible() check below returns false inside docker (without IB)

So to fix the test, we can either convert the warning to ucs_diag, or check memlock limit after line 1228.
I think we should do both.